README.md

```diff
@@ -120,9 +137,112 @@ module.exports = function (fastify, options, next) {
 npm run example:decorator -- -l info -w
 ```
 
-### Example: Shopify HMAC Query Parameter Verification
+# Default Methods
+
+## Default extractSignature Method
```

Comment:
[nit] I think we can infer these are all defaults from the Default Methods header. We can probably remove the "Default" prefix from the subHeaders. (here and elsewhere)

Reply:
Good point. I added the Default Methods header after the fact. Fixed.